website/docs/upgrades/1.0_to_1.1/automatic_schema_matching.md

```diff
@@ -27,6 +27,8 @@ You need to rename one of them. Depending on the circumstances and your preferen
 - If you control both configs, you can rename either of them.
 - If you only control the config file, rename it.
 
+You must make sure that the `ConfigStore` `name` references the schema's Structured Config and the `@hydra.main` `config_name` references the yaml config file.
+
```

Comment:
The wording here is confusing to me, and I worry that it may be misleading.
Option 2 (item 3) below is relevant to `@hydra.main`'s `config_name` keyword arg. It says this currently:
> ### Option 2: rename the config file
> ...
> 3. Update references to the config name accordingly, e.g. on the command-line `db=mysql` would become `db=custom_mysql`, and in a defaults list `db: mysql` would become `db: custom_mysql`.

Perhaps we could update it like this:
> ...
> 3. Update references to the config name accordingly, e.g. on the command-line `db=mysql` would become `db=custom_mysql`, in a defaults list `db: mysql` would become `db: custom_mysql`, and `@hydra.main(..., config_name="mysql")` would become `@hydra.main(..., config_name="custom_mysql")`.

@gabriel-hurtado, does this make sense to you?

Reply:
Perhaps a similar modification could be made to Option 1 below:
`cs.store(name="mysql", ...)` -> `cs.store(name="base_mysql", ...)`